docs/catalog/web/keep-transfer-sizes-small.md

```diff
@@ -0,0 +1,48 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: web
+tags: 
+ - size:medium
+ - role:web-developer
+ - web
+---
+
+# Keep transfer sizes small
+
+## Description
+
+Displaying a web page requires that the browser has downloaded all necessary files needed. Not only does the amount of request determine how much CO2 is used, but also the file sizes of each request.
+
+To use the least amount of CO2 it is more efficient to download a larger, completely necessary, resource instead of multiple tiny resources. 
+
+## Solution
```

Comment:
I think it's best if we split those out. Would you mind also explaining what you mean by (TODO Link to pattern)?